worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java

```diff
@@ -125,7 +133,12 @@ private AsyncCommandBuilder(Callable<T> callable, Actor sender) {
     public ListenableFuture<T> buildAndExec(ListeningExecutorService executor) {
         final ListenableFuture<T> future = checkNotNull(executor).submit(this::runTask);
         if (delayMessage != null) {
```

Comment:
does this handle the case where delayMessage == null && repeatedMessage != null? what's intended behavior there? possibly it should use the repeatedMessage at 1 second and then repeat it every 10 as well?

Reply:
It currently doesn't, but IMO the delayMessage should be required to use the repeatedMessage. So I'll just make the builder require delayMessage when repeatedMessage is passed.